### PR TITLE
Add API actions to add a single team, organization and group

### PR DESCRIPTION
**Pull request by nickygerritsen**
This uses the existing ImportExportService to be able to add single entities. You can `POST` both JSON as well as form data.

### CODE REVIEW

**Comment by nickygerritsen** [comment]
This might need additions to the documentation, although I'd say we have the API docs and these map directly to them.

If documentation is needed, I'd like to hear a proposal on what to add. And I didn't create it yet since I first wanted opinions on whether this is a good idea anyway.

**Comment by thijskh** [comment]
I don’t think we need additional documentation besides the generated API docs for straightforward CRUD endpoints.

File: webapp/src/Controller/API/GroupController.php [hunk 1]
```diff
@@ -66,6 +70,44 @@ public function singleAction(Request $request, string $id)
...
+     *         mediaType="multipart/form-data",
+     *         @OA\Schema(ref="#/components/schemas/TeamCategory")
+     *     ),
+     *     @OA\MediaType(
+     *         mediaType="application/json",
+     *         @OA\Schema(ref="#/components/schemas/TeamCategory")
+     *     )
+     * )
+     * @OA\Response(
+     *     response="200",
```
Comment (mvr320) [inline webapp/src/Controller/API/GroupController.php:95]: Why should this not be 201? https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

Comment (nickygerritsen) [reply webapp/src/Controller/API/GroupController.php:95]: 201 might make sense. We should then also set the Location header

File: webapp/src/Controller/API/AbstractRestController.php [hunk 2]
```diff
@@ -105,12 +106,20 @@ protected function performSingleAction(Request $request, string $id)
...
+     *
      * @return Response
      */
-    protected function renderData(Request $request, $data): Response
-    {
+    protected function renderData(
+        Request $request,
+        $data,
+        int $statusCode = Response::HTTP_OK,
+        array $headers = []
```
Comment (meisterT) [inline webapp/src/Controller/API/AbstractRestController.php:121]: nit: let's call this `extraHeaders` or similar?

File: webapp/src/Controller/API/AbstractRestController.php [hunk 3]
```diff
@@ -105,12 +106,20 @@ protected function performSingleAction(Request $request, string $id)
 
     /**
      * Render the given data using the correct groups
-     * @param Request $request
-     * @param mixed $data
+     *
+     * @param Request  $request
```
Comment (meisterT) [inline webapp/src/Controller/API/AbstractRestController.php:110]: Do we need the `@param` annotation here with the type hints below?

**Review by meisterT — APPROVED** [review]

File: webapp/src/Controller/API/AbstractRestController.php (hunk 3 above)
Comment (nickygerritsen) [reply webapp/src/Controller/API/AbstractRestController.php:110]: Nope will remove them

**Pull request merged**